databases/week1/hyf_lesson1.sql

```diff
@@ -0,0 +1,40 @@
+SELECT * -- get the table
+FROM task;
+SELECT COUNT(title) --how many tasks are in the task table
+FROM task;
+SELECT COUNT(id) -- count of do not have valid date
+FROM task
+WHERE due_date is NULL;
+SELECT task.title, --all the tasks that are marked as done
+status.name
+FROM task JOIN status on task.status_id = status.id 
+WHERE status.name LIKE'Done';
+SELECT task.title, -- all the tasks that are not marked as done
+status.name
+FROM task JOIN status on task.status_id <> status.id 
```

Comment:
Are you sure about the `<>`? What does it mean?

Reply:
<> it means not equal to (!=)

Reply:
Exactly, and is that what you want? 
What kind of results did you get from that query?